app/_hub/tomkerkhove/microsoft_azure_container_instances/index.md

```diff
@@ -0,0 +1,222 @@
+---
+name: Microsoft Azure Container Instances
+publisher: Tom Kerkhove
+
+categories:
+  - deployment
+
+type: integration
+
+desc: Deploy Kong on Microsoft Azure Container Instances # (required) 1-liner description; max 80 chars
+# (required) extended description.
+  # Use YAML piple notation for extended entries.
+  # EXAMPLE long text format (do not use this entry)
+  # description: |
+  #   Maintain an indentation of two (2) spaces after denoting a block with
+  #   YAML pipe notation.
+  #
+  #   Lorem Ipsum is simply dummy text of the printing and typesetting
+  #   industry. Lorem Ipsum has been the industry's standard dummy text ever
+  #   since the 1500s.
+description: |
+  This guide walks you through deploying and running Kong on Microsoft Azure Container Instances.
+
+  We will be using either Azure Database for PostgreSQL or Cosmos Db to store the gateway configuration.
+support_url: # (Optional) A specific URL of your own for this extension.
+  # Defaults to the url setting in your publisher profile.
+
+source_url: https://github.com/tomkerkhove/kong-deployment-on-azure
+  # (Optional) If your extension is open source, provide a link to your code.
+
+#license_type:
+  # (Optional) For open source, use the abbreviations in parentheses at:
+  # https://opensource.org/licenses/alphabetical
+
+#license_url:
+  # (Optional) Link to your custom license.
+
+#privacy_policy:
+  # (Optional) If you have a custom privacy policy, place it here
+
+#privacy_policy_url:
+  # (Optional) Link to a remote privacy policy
+
+#terms_of_service:
+  # (Optional) Text describing your terms of service.
+
+#terms_of_service_url:
+  # (Optional) Link to your online TOS.
+
+# COMPATIBILITY
+# In the following sections, list Kong versions as array items.
+# Versions are categorized by Kong edition and their known compatibility.
+# Unlisted Kong versions will be considered to have "unknown" compatibility.
+# Uncomment at least one of 'community_edition' or 'enterprise_edition'.
+# Add array-formatted lists of versions under their appropriate subsection.
+
+kong_version_compatibility: # required
+  community_edition: # optional
+    compatible:
+        - 0.14.x
+        - 0.13.x
+        - 0.12.x
+        - 0.11.x
+        - 0.10.x
+        - 0.9.x
+        - 0.8.x
+        - 0.7.x
+        - 0.6.x
+        - 0.5.x
+        - 0.4.x
+        - 0.3.x
+        - 0.2.x
+    #incompatible:
+  enterprise_edition: # optional
+    compatible:
+        - 0.34-x
```

Comment:
Same comment as above - if you've not confirmed compatibility, either mark these as incompatible, or comment it out for "unknown compatibility"